src/contractWrappers/ArbitrableTransactionWrapper.js

```diff
@@ -195,13 +198,93 @@ class ArbitrableTransactionWrapper extends ContractWrapper {
         {
           from: account,
           gas: config.GAS,
-          value: 0
+          value: 0,
         }
       )
 
     return txHashObj.tx
   }
 
+  /**
+  * Call by partyA if partyB is timeout
+  * @param {string} account ETH address of user
+  * @param {string} contractAddress ETH address of contract
+  * @return {string} txHash Hash transaction
+   */
+  timeOutByPartyA = async (
+    account = this._Web3Wrapper.getAccount(0),
+    contractAddress
+  ) => {
+    try {
+      this.contractInstance = await this.load(contractAddress)
+
+      const status = await this.contractInstance.status.call()
+      const timeout = await this.contractInstance.timeout.call()
+      const lastInteraction = await this.contractInstance.lastInteraction.call()
+
+      if (status != CONTRACT_ARBITRABLE_STATUS.WAITING_PARTY_B) {
+        return new Error('Status contract is not WAITING_PARTY_B')
+      }
+
+      if (Date.now() >= (lastInteraction+timeout)) {
+        return new Error('The timeout is not reached')
+      }
+
+      const txHashObj = await this.contractInstance
+        .timeOutByPartyA(
```

Comment:
I think it might be worth making a PR to the contract as well to change these names